scripts/motor-expert-screen

```diff
@@ -96,17 +96,20 @@ elif [ "${rtyp}" == 'motor' ]; then
     caget "$PREFIX:PN" > /dev/null 2>&1
     if [ $? -eq 0 ]; then
         /reg/g/pcds/package/epics/3.14//modules/pcds_motion/${OLD_VERSION}/launch-motor.sh "$PREFIX" > /dev/null 2>&1 &
-    elif [[ $PREFIX == *"MCS2"* ]]; then # smaracts have "motor" rtyp, so we catch them with the PREFIX
-        enc=$(caget $PREFIX.UEIP)
-        if [[ $enc == *"Yes"* ]]; then
-            edm -x -eolc -m "MOTOR=$PREFIX" mcs2_main.edl > /dev/null 2>&1 &
-        else
-            edm -x -eolc -m "MOTOR=$PREFIX" mcs2_openloop.edl > /dev/null 2>&1 &
-        fi
     else
-        cd /reg/neh/home/klg/epics/ioc/common/aerotech/current/motorScreens
-        #cd /reg/neh/home4/mcbrowne/trunk2/ioc/common/aerotech/current/motorScreens
-        edm -x -eolc -m "MOTOR=${PREFIX}" ens_main.edl >& /dev/null &
+	# smaracts have "motor" rtyp, so we catch them with the PREFIX or with smaract specific pvs
+	if [[ $PREFIX == *"MCS2"* || $(caget "$PREFIX:PTYPE_RBV" > /dev/null 2>&1) -eq 0 ]]; then
+            enc=$(caget $PREFIX.UEIP)
+            if [[ $enc == *"Yes"* ]]; then
+		edm -x -eolc -m "MOTOR=$PREFIX" mcs2_main.edl > /dev/null 2>&1 &
+            else
+		edm -x -eolc -m "MOTOR=$PREFIX" mcs2_openloop.edl > /dev/null 2>&1 &
+            fi
+	else
+            cd /reg/neh/home/klg/epics/ioc/common/aerotech/current/motorScreens
```

Comment:
Woah, just noticed this. Why are we running screens out of Karl's (or Mike's) home directory?

Reply:
This showed up in the diff because I nested an existing conditional block. I could add /reg/g/pcds/epics/ioc/common/aerotech/R1.2.0/motorScreens to EDMDATAFILES and use ens_main.edl without changing directories. Is that reasonable?

Reply:
Moving to a released version is best, but that's a issue un-related to your PR. I'm still a little puzzled about the way this script decides to open the aerotech screen.  I'll think about this more, and perhaps open a separate issue on this repo.

Reply:
I agree, this is a problem but it's not a new problem

Reply:
I made a new issue for this here: https://github.com/pcdshub/engineering_tools/issues/175